libr/core/rtr.c

```diff
@@ -783,7 +783,7 @@ static int r_core_rtr_http_thread (RThread *th) {
 	HttpThread *ht = th->user;
 	if (!ht || !ht->core) return false;
 	ret = r_core_rtr_http_run (ht->core, ht->launch, ht->path);
-	free (ht->path);
+	free ((void *)ht->path);
```

Comment:
free() doesn't like const char * arguments. I can cast to char* if you prefer.
The original code didn't strdup(path) when it passed it to the thread which looks like a race condition to me.